### PR TITLE
fix(DDI-1327): Updated angular code sample

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Updated angular code under Basic example of File Upload component

### CODE REVIEW

**Review by ArakTaiRoth — DISMISSED** [review]

File: libs/docs/src/components/common/FileUpload.stories.mdx [hunk 1]
```diff
@@ -170,8 +170,17 @@ export const UploadTemplate = (props) => {
...
       code={`
+      interface Uploader {
+        upload: (url: string | ArrayBuffer) => void;
+        abort: () => void;
+      }
+      interface Upload {
+        file: File;
+        uploader: Uploader;
+      }
+      class MockUploader implements Uploader { // implement your logic to upload files}
```
Comment (vanessatran-ddi) [inline libs/docs/src/components/common/FileUpload.stories.mdx:181]: I think the } is ignored here. 
image

Suggestion:
```
class MockUploader implements Uploader {
// implement...
}
```
so it is easier to understand on the storybook.

**Review by vanessatran-ddi — CHANGES_REQUESTED** [review]

File: libs/docs/src/components/common/FileUpload.stories.mdx (hunk 1 above)
Comment (Radha-Alberta) [reply libs/docs/src/components/common/FileUpload.stories.mdx]: Changed.

**Review by vanessatran-ddi — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

File: libs/docs/src/components/common/FileUpload.stories.mdx [hunk 2]
```diff
@@ -170,8 +170,19 @@ export const UploadTemplate = (props) => {
...
+      interface Upload {
+        file: File;
+        uploader: Uploader;
+      }
+      class MockUploader implements Uploader {
+        // implement your logic to upload files
+      }
       export class FileUploadComponent {
-        uploads: Upload[] = [];
+        uploads: Upload[] = []; <!-- Upload is customized according to project needs, it should have the file property -->
```
Comment (chrisolsen) [inline libs/docs/src/components/common/FileUpload.stories.mdx:185]: The comments added to the snippets or in html format, not the js/ts that should probably be used

**Review by chrisolsen — CHANGES_REQUESTED** [review]

File: libs/docs/src/components/common/FileUpload.stories.mdx (hunk 2 above)
Comment (Radha-Alberta) [reply libs/docs/src/components/common/FileUpload.stories.mdx]: Changed the comments

**Review by ArakTaiRoth — APPROVED** [review]

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**